documentation/adr/0012-matlab-python-wrapper.md

```diff
@@ -0,0 +1,53 @@
+[<-previous](0011-use-mex-for-pyHorace-python-calls.md) | [next->]()
+
+# 12. A Matlab wrapper for Python PACE projects
+
+Date: 2021-Jan-01
+
+## Status
+
+Proposed
+
+
+## Context
+
+Both Euphonic and Brille are PACE projects with primarily a Python interface.
+PACE, however, aims to provide both a Python and a Matlab interface to users,
+and also to foster inter-operability between projects which are written both in Matlab and Python.
+In particular, `pyHorace` ([prototype](https://github.com/mducle/hugo)), cannot use the standard
+method for Matlab to run Python code, using the `py.*` namespace.
```

Comment:
It's worth stating what the [standard method is](https://www.mathworks.com/help/matlab/call-python-libraries.html) and why `pyHorace` can't use it. It's alluded to below with the mention of crashes but could do with being explicit.